notebooks/getting_started/part3_exploring_cohorts.ipynb

```diff
@@ -1,15 +1,5 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    image_series_UID = os.listdir(f"IDC_sorted/{patient_ID}/{study_UID}/CT")[0] # There is only one series with CT data
</pre><p><code>study_UID</code> is undefined here </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ImagingDataCommons/IDC-Tutorials/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>It was defined a few cells previous, no issues when running this notebook!</p>